README.md

```diff
@@ -83,6 +83,55 @@ In the index.html, include the core branding stylesheet:
 #### **avatar**
 `string` Avatar image URL. Default is the special provided Antwerp icon.
 
+### Events
+
+#### **actionClicked**
+Triggers when the user clicks a button of a message with type `action`. This type of message can only be generated by the BFF package. The `actionClicked` output event requires a defined action and pauses the workings of the chatbot widget, waiting for a method call (`completeAction`) that has the same defined action. It allows the developer to handle some things outside the chatbot (e.g. a payment) before returning to the chatbot conversation.

```

Comment:
This type of message is injected by the BFF by injecting it into the response. The package itself doesn't have the ability to modify a response.

Reply:
Should this be "actionStarted"? It matches up with "completeAction", and we may at some point trigger actions from something else besides a button.

Reply:
Fixed.